### PR TITLE
Include links that help one "translate" the outdated PureScript book

### LINKED ISSUE #224: Add more context to PureScript by Example book's outdated nature
See purescript/purescript.github.io#133 for full context.

Tasks
- Forewarn about the PureScript by Example book's code being outdated
- Include fork to dwhitney's code for the Book.

### PR DESCRIPTION
**Pull request by JordanMartinez**
Fixes #224 

Note: I also added links to Justin's `0.11.7` to `0.12.x` page and a list of deprecated packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ Feel free to make an issue to discuss amending the scope.
 
 ## Learning
 
-- The [PureScript Book](https://leanpub.com/purescript/read) is the recommended approach to learning the language, since it covers more material in greater depth. However, it is not updated yet for the 0.12 version of the compiler.
+- The [PureScript Book](https://leanpub.com/purescript/read) is the recommended approach to learning the language, since it covers more material in greater depth. However, it covers `0.11.7` and is not updated yet for the `0.12.x` version of the compiler. Thus, one should be aware of the following materials when reading through the book:
+    - See [dwhitney's fork of the book's exercises](https://github.com/dwhitney/purescript-book) and the other materials in this section for supplemental reading.
```
Comment (chexxor) [inline README.md]: It's not mentioned here that the usefulness of dwhitney's fork is that the code samples have been updated to work with 0.12.x version of the compiler -- I think it probably should. I could add a commit on top of this later, too, if you don't want to change it before merging.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,10 @@ Feel free to make an issue to discuss amending the scope.
...
+    - See [Justin's `0.11.7` to `0.12.x` summary](https://purescript-resources.readthedocs.io/en/latest/0.11.7-to-0.12.0.html) to know how to 'translate' the outdated book's code into working code.
+    - be wary of any references to these [deprecated packages](https://github.com/purescript-deprecated) in the book.
```
Comment (chexxor) [inline README.md]: `be` -> `Be`

**Comment by chexxor** [comment]
LGTM otherwise. 👍

**Comment by JordanMartinez** [comment]
I addressed your comments. I also removed the redundant comment from when I was initially adding things.

**Comment by chexxor** [comment]
Cool, I'll merge in several hours.

**Pull request merged**